src/openalea/lpy/parameters/schema/lpy.json

```diff
@@ -0,0 +1,42 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "L-Py",
+    "type": "object",
+    "required": [
```

Comment:
credits in not required but this gives a keyerror

```py
lp = LsystemParameters()
lp.loads('{"schema": "lpy", "version": "1.0", "options": {}, "materials": [], "parameters": []}')

```

Reply:
Ok, I changed the code so that only schema and version are required.